README.md

```diff
@@ -9,8 +9,8 @@ jsk openni tools
 
 ### Create catkin workspace
 ```
-$ mkdir-p catkin_ws/ws_jsk_openni_kinect/src
-$ cd catkin_ws/ws_jsk_openni_kinect
+$ mkdir-p ~/catkin_ws/ws_jsk_openni_kinect/src
```

Comment:
you can create PR against existing PR https://github.com/jsk-ros-pkg/jsk_openni_kinect/pull/18